lib/floe/workflow/state.rb

```diff
@@ -29,9 +29,32 @@ def initialize(workflow, name, payload)
         @comment  = payload["Comment"]
       end
 
+      def run!(input)
+        run_async!(input)
+        sleep(1) while running?
```

Comment:
almost feel like dropping `State#run!` and moving that blocking knowledge into `Workflow` may make sense. thought this concept is pretty simple

Reply:
Yeah I think you're right not sure what this buys us since no one should be hitting the state directly.  Only callers right now are in state specs.